### ISSUE TITLE
Fedora 21: novm rpm install path conflict with filesystem package

### ISSUE THREAD

**Issue by tjyang** [google/novm#12]
```
[tjyang@fedora21 novm]$ sudo rpm -ivh novm-0.0-164.gb7c9c78.x86_64.rpm 
Preparing...                          ################################# [100%]
    file /usr from install of novm-0.0-164.gb7c9c78.x86_64 conflicts with file from package filesystem-3.2-28.fc21.x86_64
    file /usr/bin from install of novm-0.0-164.gb7c9c78.x86_64 conflicts with file from package filesystem-3.2-28.fc21.x86_64
    file /usr/lib from install of novm-0.0-164.gb7c9c78.x86_64 conflicts with file from package filesystem-3.2-28.fc21.x86_64
[tjyang@fedora21 novm]$ 

```

**Comment by foxwang** [comment]
hit it 。
centos7

**Comment by amscanne** [comment]
I've merged #14, which hopefully should fix it. Reopen if it doesn't.
